h/templates/help.html.jinja2

```diff
@@ -34,19 +48,22 @@
           <div class="numbered-list columns">
             <div class="columns__col-1-2"><div class="numbered-list-item">If you want to annotate and comment on documents then install our <a href="{{ request.route_url('chrome-extension') }}">browser extension</a>.</div></div>
             <div class="columns__col-1-2"><div class="numbered-list-item">If you wish to install Hypothesis on your own site then head over to GitHub.</div></div>
+        <div class="wrap-columns">
+          <div class="wrap-columns__col">
+            <img
+              alt="Select the puzzle-piece icon (extension menu) in the Chrome location bar, then 'pin' the Hypothesis extension so it is always available"
+              src="{{ asset_url('images/chrome-pin-extension-01.png') }}"
+            />
```

Comment:
`<img>` elements are self closing, so no need for the `/>` in HTML. I couldn't reproduce the issue you mentioned with the `alt` attribute being "eaten". The `alt` attribute appeared just fine in the DOM inspector in my version of Chrome and was visible via the `alt` property of the element.

Reply:
How weird...I swear...well, anyway.